README.rst

```diff
@@ -80,6 +80,50 @@ in search mode is provided below:
         bind_password: "ch33kym0nk3y"
         #filter: "(objectClass=posixAccount)"
 
+Active Directory forest accounts are also supported with custom
+separator (`=` or `/`) in the following form: `login<separator>domain`.
+
+Please note that common domain separators `@` and `\\` can not be used in Matrix
+User Indentifiers (https://matrix.org/docs/spec/appendices#user-identifiers)
+and `/` separator is used by default.
+
+Let's say you have several domains in example.com forest:
```

Comment:
I think this could be reworded a bit.

```suggestion
If the ``ad_forest`` flag is set to ``true``, an Active Directory forest will be
searched for the login details.

In this mode, the user enters their login details in the form ``<login>/<domain>``. (The 
normal domain separators, ``@`` and ``\\``, cannot be used in Matrix User Identifiers, so 
``/`` is used by default instead.)
```